yellowbrick/base.py

```diff
@@ -407,3 +408,119 @@ def predict(self, X, y):
         """
         for model in self.models:
             yield cvp(model, X, y, cv=12)
+
+
+
+#TODO it appears that the previous class (MultiModelMixin) is not actually used
+# anywhere and would be redundant to this new class. Remove that one?
+class MultipleVisualizer(Visualizer):
+    """
+    Used as a base class for visualizers that use subplots.
+
+    Parameters
+    ----------
+    visualizers : A list of instantiated visualizers
+
+    nrows: integer, default: None
+        The number of rows desired, if you would like a fixed number of rows.
+        Specify only one of nrows and ncols, the other should be None. If you 
+        specify nrows, there will be enough columns created to fit all the 
+        visualizers specified in the visualizers list. 
+    
+    ncols: integer, default: None
+        The number of columns desired, if you would like a fixed number of columns.
+        Specify only one of nrows and ncols, the other should be None. If you 
+        specify ncols, there will be enough rows created to fit all the 
+        visualizers specified in the visualizers list. 
+
+    axarr: matplotlib.axarr, default: None.
+        If you want to put the plot onto an existing axarr, specify it here. Otherwise a new
+        one will be created. 
+
+    kwargs : additional keyword arguments, default: None
+        Any additional keyword arguments will be passed on to the fit() method and therefore 
+        passed on to the fit() method of the wrapped estimators, if applicable. Otherwise ignored.
+
+    Examples
+    --------
+    >>> from yellowbrick.base import MultipleVisualizer
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from yellowbrick.classifier import ConfusionMatrix
+    >>> from yellowbrick.classifier import ClassBalance
+    >>> model = LogisticRegression()
+    >>> visualizers = [ClassBalance(model),ConfusionMatrix(model)]
+    >>> mv = MultipleVisualizer(visualizers, ncols=2)
+    >>> mv.fit(X_train, y_train)
+    >>> mv.score(X_test, y_test)
+    >>> mv.poof()
+    """
+    def __init__(self, visualizers = [], nrows = None, ncols = None, axarr = None, **kwargs):
+        self.visualizers = visualizers
+        self.plotcount = len(visualizers)
```

Comment:
When we do get around implementing the flipbook visualizer, I intend to make a superclass for both this class and that one that treats the visualizers as an immutable collection, subclassing [`collections.Sequence`](https://docs.python.org/3/library/collections.abc.html#collections.abc.Sequence) -- which requires implementing `__getitem__` and `__len__` on the class; which are pretty straightforward since they simply reference the underlying visualizers list. 

This has two important points for this code:

1. We should store the visualizers as `self._visualizers` then create a visualizers property with no setter (e.g. a clear signal you can't change the visualizers after instantiation). 
2. We should not store `self.plotcount`; as it's only used here, I'd recommend just using a local variable, `plotcount` 

If you'd like to implement this as a sequence straight away, please feel free!

Reply:
I've switched this to an @property. Instead of not creating a setter, I created one that just raises an AttributeError - not sure if that's best practice but seemed even more clear by raising the error if the user tries to edit the visualizers list.

Reply:
If you leave the setter off and try to assign a value to the property an `AttributeError: can't set attribute` is raised. Your exception is equivalent and more informative, so I'm happy to go with that, though I think it's more typical to simply leave the setter off. 

```python
class Thing(object):

    def __init__(self, x):
        self._x = x 

    @property
    def x(self):
        return self._x

>>> t = Thing(42)
>>> t.x = 'bob'
Traceback (most recent call last):
  File "t.py", line 13, in <module>
    t.x = 'bob'
AttributeError: can't set attribute
```